apps/app/src/test/scala/org/lfdecentralizedtrust/splice/integration/tests/ValidatorProxyIntegrationTest.scala

```diff
@@ -0,0 +1,73 @@
+package org.lfdecentralizedtrust.splice.integration.tests
+
+import com.digitalasset.canton.discard.Implicits.DiscardOps
+import org.lfdecentralizedtrust.splice.integration.EnvironmentDefinition
+import org.lfdecentralizedtrust.splice.integration.tests.SpliceTests.IntegrationTest
+import org.lfdecentralizedtrust.splice.util.WalletTestUtil
+import org.lfdecentralizedtrust.splice.unit.http.{
+  SystemProperties,
+  SystemPropertiesSupport,
+  TinyProxySupport,
+}
+
+class ValidatorProxyIntegrationTest
+    extends IntegrationTest
+    with WalletTestUtil
+    with TinyProxySupport
+    with SystemPropertiesSupport {
+
+  override def environmentDefinition: SpliceEnvironmentDefinition =
+    EnvironmentDefinition
+      .simpleTopology1Sv(this.getClass.getSimpleName)
+      .withManualStart
+
+  "validator should start and tap, using http forward proxy without proxy authentication" in {
+    implicit env =>
+      val host = "127.0.0.1"
+      withProxy() { proxy =>
+        val props =
+          SystemProperties()
+            .set("https.proxyHost", "localhost")
+            .set("https.proxyPort", proxy.port.toString)
+            // localhost and tcp loopback addresses are not proxied by default in gRPC and Java URL connections
+            // so we need to override the nonProxyHosts to ensure our proxy is used for all connections
+            .set("http.nonProxyHosts", "")
+            .set("https.nonProxyHosts", "")
+        withProperties(props) {
+          initDsoWithSv1Only()
```

Comment:
Hmm, shouldn't we start this without the proxy? And wrap with the proxy just the alice validator start. That way we ensure that the assertions we do in the proxy were done by the validator and not the SV

Reply:
(this was also fixed in the end)